src/compas_rhino/install.py

```diff
@@ -16,8 +16,10 @@
     'install',
     'installable_rhino_packages',
     'after_rhino_install',
+    'INSTALLED_VERSION'
```

Comment:
I would not make it part of the official API, it's more an internal thing, so, I'd remove it from `__all__`. Other than that, LGTM!

Reply:
agreed about the public API thing

Reply:
makes sense! adjusted now